components/MoreInfoForm.js

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function MoreInfoForm({
+  personId,
+  onUpdateEntryNotes,
+  currentProfile,
+  notes,
+}) {
+  const [edit, setEdit] = useState(false);
+
+  function handleNotesSubmit(event) {
+    event.preventDefault();
+    const adaptedNotes = event.target.adaptedNotes.value;
+    onUpdateEntryNotes(adaptedNotes, personId);
+    setEdit(!edit);
+  }
+
+  return (
+    <>
+      <StyledInfo>Notizen zur Person</StyledInfo>
+      <StyledNotesWrapper>
+        {edit === true || !notes ? (
```

Comment:
meeeega 🥳 kleiner Vorschlag noch: 

```suggestion
        {edit === true || !currentProfile.notes ? (
```